frontend/src/customPropTypes/mines.js

```diff
@@ -26,3 +26,11 @@ export const minePageData = shape({
   total: PropTypes.numnber,
   total_pages: PropTypes.numnber,
 });
+
+export const mineName = shape({
```

Comment:
`mineName` is very specific for an object that contains more than a name. This should be renamed.

Reply:
`mineNameList: getMineNames(state).mines,` existing places that used that object structure use a selector called getMineNames, so i figure the object type should be MineName. I don't disagree and I'd be happy to rename it. But then the selector name and the customproptype don't match in name, which may be awkward.

Reply:
ah, okay, I'm following. 

This is fine then.